README.md

```diff
@@ -5,7 +5,9 @@ This is an application to list all the Java Developers in Nairobi as provided by
 [![Test Coverage](https://api.codeclimate.com/v1/badges/fe42defb6a5be4f552bb/test_coverage)](https://codeclimate.com/github/petermwash/JavaDevsNairobi/test_coverage)
 [![Maintainability](https://api.codeclimate.com/v1/badges/fe42defb6a5be4f552bb/maintainability)](https://codeclimate.com/github/petermwash/JavaDevsNairobi/maintainability)
 
-# Screenshots for mockups
+# Screenshots for mock-ups
```

Comment:
@petermwash 
Please, could you consider separating concerns because this is updating the README yet the task is about **layout**.

Or you could consider explicitly mentioning this in your PR description.

Reply:
@wasswa-derick 
Thanks for pointing that out, I have mentioned on the *background context* of my PR.